app/bundles/EmailBundle/Entity/Email.php

```diff
@@ -984,6 +992,24 @@ public function setPlainText($plainText)
         return $this;
     }
 
+    public function isContinueSending(): ?bool
```

Comment:
I always like when the code reads like a real language (English) and this doesn't read good. That's what I like on Laravel. How about this?
```suggestion
    public function shouldContinueSending(): bool
```

Reply:
This wasn't addressed. Please do not resolve the conversation that is not addressed.